lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ @use '../../scss-shared/nx-header-variables';
+
+ // 74px refers to the total height of each toast: the content itself, plus the padding
+ // and margin-top applied. 
+ $-nx-toast-height: 74px;
+
+.nx-toast-container {
+  grid-area: content;
+  justify-self: end;
+  margin-right: var(--nx-spacing-2x);
+  width: 500px;
+  z-index: var(--nx-z-index-toast);
+  overflow: hidden;
+  pointer-events: none;
+}
+
+// in legacy layouts where page scrolling is enabled, position: fixed helps keep
+// NxToastContainer in place. Top refers to the set height of the the header
+.nx-html--page-scrolling {
+  .nx-toast-container {
+    position: fixed;
+    top: nx-header-variables.$main-header-height;
+    right:0;
+  }
+}
+
+.nx-toast {
+  position: relative;
+  animation: animationIn 0.4s linear;
+  right: 0;
+  opacity: 1;
+  margin-top: var(--nx-spacing-2x);
+  overflow: hidden;
+  pointer-events: auto;
+
+  &.nx-toast--closing {
+    animation-name: animationOut;
+  }
+
+  .nx-alert {
```

Comment:
Let's add use the `container-vertical` mixin on `.nx-toast` which will allow us to avoid adding custom styles to `.nx-toast .nx-alert` entirely.

Reply:
> Let's use the `container-vertical` mixin on `.nx-toast` which will allow us to avoid adding custom styles to `.nx-toast .nx-alert` entirely.

mixin added, and custom styling on nx-alert removed here: [251c518](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/251c518f75cdfe5952c1f4f71346111491d3add8)